app/controllers/api/people/search_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
     private
 
     def search_results
-      PeopleSearch.new(query).entries
+      PeopleSearch.new(query, false).entries
```

Comment:
I know this is just a fix for code that is probably not used anymore but you could give this variable a name when passing it, just `false` is quite confusing :grin: 

`PeopleSearch.new(query, search_skills: false).entries`